static/js/components/ImageAnalysisForm.jsx

```diff
@@ -0,0 +1,215 @@
+
+  const handleSubmit = async ({ formData }) => {
+    const data = { ...formData };
+    data.instrument_id = selectedInstrumentId;
+    data.obstime = data.obstime.replace("+00:00", "").replace(".000Z", "");
+    console.log(data.image_data);
+
+    const result = await dispatch(
+      sourceActions.submitImageAnalysis(obj_id, data)
+    );
+    if (result.status === "success") {
+      dispatch(showNotification("Image analysis submitted"));
+    }
+  };
+
+  if (enum_types.length === 0) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+
+  const telLookUp = {};
+  // eslint-disable-next-line no-unused-expressions
+  telescopeList?.forEach((tel) => {
+    telLookUp[tel.id] = tel;
+  });
+
+  const instLookUp = {};
+  // eslint-disable-next-line no-unused-expressions
+  instrumentList?.forEach((instrumentObj) => {
+    instLookUp[instrumentObj.id] = instrumentObj;
+  });
+
+  const handleSelectedInstrumentChange = (e) => {
+    setSelectedInstrumentId(e.target.value);
+  };
+
+  const catalogs = [
+    "ps1",
+    "gaiadr2",
+    "gaiaedr3",
+    "skymapper",
+    "vsx",
+    "apass",
+    "sdss",
+    "atlas",
+    "usnob1",
+    "gsc",
+  ];
+  const astrometric_refinement_method = ["scamp", "astropy", "astrometrynet"];
+  const templates = ["PanSTARRS/DR1/g", "PanSTARRS/DR1/r", "PanSTARRS/DR1/i"];
+
+  const imageAnalysisFormSchema = {
+    type: "object",
+    properties: {
+      obstime: {
+        type: "string",
+        format: "date-time",
+        title: "Image Date [UTC]",
+        default: defaultDate,
+      },
+      filter: {
+        type: "string",
+        oneOf: instLookUp[selectedInstrumentId]?.filters.map((filter) => ({
+          enum: [filter],
+          title: `${filter}`,
+        })),
+        title: "Filter list",
+        default: instLookUp[selectedInstrumentId]?.filters[0],
+      },
+      gain: {
+        type: "number",
+        title: "Gain",
+        default: instLookUp[selectedInstrumentId]?.gain,
+      },
+      s_n_detection: {
+        type: "number",
+        title: "S/N Detection",
+        default: 5,
+      },
+      s_n_blind_match: {
+        type: "number",
+        title: "S/N Blind Match",
+        default: 20,
+      },
+      upper_scale: {
+        type: "number",
+        title: "Upper Scale",
+        default: 5,
+      },
+      lower_scale: {
+        type: "number",
+        title: "Lower Scale",
+        default: 0.1,
+      },
+      astrometric_refinement_cat: {
+        type: "string",
+        title: "Astrometric Refinement Catalog",
+        default: "ps1",
+        enum: catalogs,
+      },
+      astrometric_refinement_meth: {
+        type: "string",
+        title: "Astrometric Refinement Method",
+        default: "astropy",
+        enum: astrometric_refinement_method,
+      },
+      matching_radius: {
+        type: "number",
+        title: "Matching Radius",
+        default: 2 / 3600,
+      },
```

Comment:
@mcoughlin what do you think?

Reply:
@Theodlz I agree. Just make sure to divide through by 3600 when passing the radius (so it is in degrees). and add arcseconds to the label here.

Reply:
Ok! But should we divide by 3600 in the frontend before sending it to the API call, or tell users: "ok submit this in arcsec, both from the frontend and for API calls" and we divide it in the backend? 

Because if we divide it by 3600 (meaning we expect arsec) only in the frontend, that means that they need to use degree instead for API calls, which seems weird to me right? Submitting 2 different things if you are using the frontend or the API is quite confusing.

Reply:
Oh and also, we don't use most of the params of the frontend form in the backend yet. Like we don't use that radius in the handler for instance.

Reply:
Should we take care of that in that PR?

Reply:
@mcoughlin

Reply:
I think we should use arcseconds on both front and back-end. And if it will be used by for now-commented code, I think fine to have it.